### PR TITLE
docs - low memory query config for RQs and RGs

### PR DESCRIPTION
**Pull request by lisakowen**
- resource queues - added section to the best practices RQ memory/resource page and the "using resource queues" page
- resource groups - add section to the "using resource groups" page (there is currently no best practices info for RGs)

### CODE REVIEW

**Comment by pivotal-ning-yu** [comment]
Looks perfect.

File: gpdb-doc/dita/best_practices/workloads.xml [hunk 1]
```diff
@@ -56,6 +59,14 @@ VM Protect failed to allocate 4096 bytes, 0 MB available</codeblock></p>
         number of segments per host, then the amount of memory allocated in
           <codeph>gp_vmem_protect_limit</codeph> can be raised.</p>
     </section>
+    <section id="section113x" xml:lang="en">
+      <title>Low Memory Queries</title>
+        <p>A <codeph>statement_mem</codeph> setting in the 1MB range has been shown
```
Comment (dyozie) [inline gpdb-doc/dita/best_practices/workloads.xml]: Seems like you should also call out the `memory_spill_ratio=0` recommendation here.  Previous topics might want to cross-reference each other as well, if we've identified these 2 settings as applicable to low-memory queries.

Comment (lisakowen) [reply gpdb-doc/dita/best_practices/workloads.xml]: @dyozie - this best practices page is qualified/titled as applicable to resource queues only.  do you still want me to add the resource group info?

File: gpdb-doc/dita/admin_guide/workload_mgmt.xml [hunk 2]
```diff
@@ -239,6 +239,14 @@
         unbounded amounts of memory.</p>
       <p>For more information on configuring memory limits on a resource queue, and other memory
         utilization controls, see <xref href="#topic12" type="topic" format="dita"/>.</p>
+    <section id="topic113x" xml:lang="en">
+      <title>statement_mem and Low Memory Queries</title>
```
Comment (gaos1) [inline gpdb-doc/dita/admin_guide/workload_mgmt.xml]: I think this again. The description might be confusing to end users that only 1M (resource queue) or 0 (resource group) can help on low memory queries.  the fact is that smaller setting can help always help, 1M and 0 are just examples. considering that some short queries only need 2M memory, a setting statement_mem to 2M or 3M (or similar settings on memory_spill_ratio) are much better than a setting like 20M or 200M.

Comment (gaos1) [reply gpdb-doc/dita/admin_guide/workload_mgmt.xml]: @lisakowen

**Pull request merged**